tests/src/test/scala/org/http4s/parser/RangeParserSpec.scala

```diff
@@ -27,14 +28,34 @@ class RangeParserSpec extends Http4sSpec {
         Range(RangeUnit.Bytes, SubRange(0, 500)),
         Range(RangeUnit.Bytes, SubRange(0, 499), SubRange(500, 999), SubRange(1000, 1500)),
         Range(RangeUnit("page"), SubRange(0, 100)),
-        Range(10),
+        Range(10), // renderValue implementation is incorrect according to rfc7233 so this fails
```

Comment:
I would rather fix the bug than preserve bug compat. Nobody should be depending on these incorrect semantics.

Reply:
Should I just fix `renderValue` then? I can run the tests but it's hard for me to be sure I'm not breaking stuff.

Reply:
Yes, I would go ahead and fix the bug, and if tests break because they were testing broken renderings, fix those, too.  We'll be thoughtful about tests we change, but we shouldn't continue making any buggy assertions.